site/pt/blog/new-in-devtools-102/index.md

```diff
@@ -28,126 +27,156 @@ draft: true
 
 <!-- Continuing to execute JavaScript is causing a lot of trouble for developers and might leave the renderer in a broken state. This change aligns the debugging behavior with other browsers like Firefox. -->
+Continuar executando JavaScript está causando muitos problemas para devs e pode deixar o renderizador quebrado. Essa mudança alinha o comportamento do debugger com outros browsers, como o Firefox.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/QBv59pX5TE9c7iJAB3Xu.png", alt="Improved reloading at breakpoint", width="800", height="566" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/QBv59pX5TE9c7iJAB3Xu.png", alt="Melhorias no reload em breakpoints", width="800", height="566" %}
 
 {# https://chromium.googlesource.com/chromium/src/+/ea207cee9bbd9b6731228d94778b23138373ec97 #}
 
-Chromium issues: [1014415](https://crbug.com/1014415), [1004038](https://crbug.com/1004038), [1112863](https://crbug.com/1112863), [1134899](https://crbug.com/1134899)
+Issues relacionadas: [1014415](https://crbug.com/1014415), [1004038](https://crbug.com/1004038), [1112863](https://crbug.com/1112863), [1134899](https://crbug.com/1134899)
 
 
 <!-- ## Console updates  {: #console } -->
+## Atualizações no Console {: #console }
 
 <!-- ### Handle script execution errors in the Console {: #errors } -->
+### Tratamento de erros de execução de script no Console {: #errors }
 
 <!-- Errors during script evaluation in the Console now generate proper error events that trigger the `window.onerror` handler and are dispatched as `"error"` events on the window object. -->
+Erros durante a execução de um script no Console agora geram os eventos corretos de erro que ativam a chamada `window.onerror` e são enviados como eventos do tipo `"error"` para o objeto `window`.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/gBtY4zD39SPizfcCGJJW.png", alt="Handle script execution errors in the Console", width="800", height="487" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/gBtY4zD39SPizfcCGJJW.png", alt="Tratamento de erros de execução de script no Console", width="800", height="487" %}
 
 {# https://chromium.googlesource.com/v8/v8/+/56cfdd68c731c53d016326b890b56b5c30098998 #}
 
-Chromium issue: [1295750](https://crbug.com/1295750)
+Issue relacionada: [1295750](https://crbug.com/1295750)
 
 
 <!-- ### Commit live expression with Enter {: #live-expression } -->
-
+### Enter para enviar uma *live expression*  {: #live-expression }
 <!-- Once you finish typing a [live expression](/blog/new-in-devtools-70/#watch), you can click `Enter` to commit it. Previously, hitting Enter resulted in adding new lines. This is inconsistent with other parts of the DevTools.  -->
-
+Quando você terminar de digitar uma [live expression](/blog/new-in-devtools-70/#watch), você pode apertar `Enter` para enviá-la. Anteriormente, apertar `Enter` resultava na adição de novas linhas. Isso é inconsistente com outras partes do DevTools.
```

Comment:
Quando você terminar de digitar uma [live expression](/blog/new-in-devtools-70/#watch), você pode apertar `Enter` para enviá-la. Anteriormente, apertar `Enter` resultava na adição de novas linhas. Isso era inconsistente com outras partes do DevTools.

Reply:
```suggestion
Quando você terminar de digitar uma [live expression](/blog/new-in-devtools-70/#watch), você pode apertar `Enter` para enviá-la. Anteriormente, apertar `Enter` resultava na adição de novas linhas. Isso era inconsistente com outras partes do DevTools.
```